README.md

```diff
@@ -1,84 +1,147 @@
 
-- **Monitor** the predictions of deployed models.
-- **Label** data with a novel search-guided, iterative workflow.
-- **Iterate** on ****ground-truth**** and predictions to debug, track and improve your data and models over time.
-- **Build** custom ****dashboards**** on top of your model predictions and labels.
+**Ingredients**:
 
-Rubrix is composed of:
+- A zero-shot classifier from the 🤗 Hub: `typeform/distilbert-base-uncased-mnli`
+- A dataset containing news
+- A set of target categories: `Business`, `Sports`, etc.
 
-- a **Python library** to bridge data and models, which you can install via `pip`.
-- a **web application** to explore and label data, which you can launch using Docker or directly with Python.
+**What are we going to do**:
 
+1. Make predictions and log them into a Rubrix dataset.
+2. Use Rubrix UI for filtering records predicted as `Business` with high probability. Use the bulk-labeling feature for labeling 15 records as `Business`.
+3. Load the annotated examples and create a training set, which you can then use to train a supervised classifier.
 
+### 1. Predict and log
 
-This is an example for logging model predictions from a 🤗 transformers text classification pipeline:
+Let's load the zero-shot pipeline and the dataset (we are using the AGNews dataset for demonstration, but this could be your own dataset). Then, let's go over the dataset records and log them using `rb.log()`. This will create a Rubrix dataset, accesible from the UI.
 
 ```python
 from transformers import pipeline
 from datasets import load_dataset
 import rubrix as rb
 
-model = pipeline('zero-shot-classification', model="typeform/squeezebert-mnli")
+model = pipeline('zero-shot-classification', model="typeform/distilbert-base-uncased-mnli")
 
 dataset = load_dataset("ag_news", split='test[0:100]')
 
-# Our labels are: ['World', 'Sports', 'Business', 'Sci/Tech']
-labels = dataset.features["label"].names
+labels = ['World', 'Sports', 'Business', 'Sci/Tech']
 
 for record in dataset:
     prediction = model(record['text'], labels)
 
     item = rb.TextClassificationRecord(
         inputs=record["text"],
-        prediction=list(zip(prediction['labels'], prediction['scores'])),
-        annotation=labels[record["label"]]
+        prediction=list(zip(prediction['labels'], prediction['scores']))
     )
 
-    rb.log(item, name="ag_news_zeroshot")
+    rb.log(item, name="news_zeroshot")
 ```
 
-# Quick links
+### 2. Explore, Filter and Label
+
+Now we can access our Rubrix dataset and start annotating data. Let's filter the records predicted as `Business` with high probability and use the bulk-labeling feature for labeling 15 records as `Business`:
+
+https://user-images.githubusercontent.com/1107111/132261244-b9151571-608e-4a41-8f34-e9dc1c8b8e38.mp4
+
+
+### 3. Load and create a training set
+After a round of data annotation, we can load the Rubrix dataset and transform it into a 🤗 dataset for training a supervised classifier.
```

Comment:
Not sure if the transformation to a HF dataset is neccessary, maybe other frameworks prefer DataFrames. I would just explain the steps to filter the annotated records and how to extract the text + label

```suggestion
After a round of data annotation, we can load the Rubrix dataset and create a training set to train a supervised classifier.
```

Reply:
Yes, I agree!